sale_order_service/models/sale_order.py

```diff
@@ -18,8 +18,15 @@ class SaleOrder(models.Model):
     # service data
     plate_number = fields.Char(string="Plate")
     service_number = fields.Integer(string="Service number")
-    service_date = fields.Date(string="Service date")
+    service_date = fields.Datetime(string="Service date")
     origin = fields.Char(string="Origin")
     destination = fields.Char(string="Destination")
     round_trip_code = fields.Integer(string="Round trip code")
     return_service = fields.Boolean(string="Return service")
+    service_key = fields.Char(string="Service key")
+    service_transfer_reason = fields.Char(string="Service transfer reason")
+    service_invoice_number = fields.Char(string="Service invoice number")
+    service_invoice_creation_date = fields.Date(string="Service invoice creation date")
+    service_code = fields.Char(string="Service code")
+    service_name = fields.Char(string="Service name")
+    service_type = fields.Char(string="Service name")
```

Comment:
```suggestion
    service_insurer_code = fields.Char(string="Service insurer code")
    service_insurer_name = fields.Char(string="Service insurer name")
```
What is this "service_type", it seems "Tipo de servicio" which is the result of the mapping between "Clave" and "Motivo_de_Traslado",this is not stored, it should computed on-the-fly on xlsx generation.